cockatrice/src/carddatabasemodel.h

```diff
@@ -12,7 +12,7 @@ class FilterTree;
 class CardDatabaseModel : public QAbstractListModel {
     Q_OBJECT
 public:
-    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, CMCColumn };
+    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, ColorColumn };
```

Comment:
@tooomm I'm not sure I understand the objection to adding more columns? You can just scroll sideways, you won't even see this unless you want to. What's the problem here?